modules/persistent-storage-csi-sc-multiple-none.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-sc-manage
+//
+
+.Absent or multiple default storage classes
+
+Multiple default storage classes can occur if you mark a non-default storage class as default and do not unset the existing default storage class, or you create a default storage class when a default storage class is already present. In this case, any persistent volume claim (PVC) requesting the default storage class (`pvc.spec.storageClassName`=nil) gets the the most recently created default storage class, and the administrator receives an alert that there are multiple default storage classes.
+
+There are two possible scenarios where PVCs can attempt to use a non-existent default storage class:
```

Comment:
This is beta / tech preview, it won't behave this way by default as it is behind a feature gate, please mention it. @RomanBednar should be able to tell how to enable it.